modules/apps/object/object-rest-test/src/testFunctional/macros/ObjectDefinitionAPI.macro

```diff
@@ -148,6 +148,28 @@ definition {
 		return ${curl};
 	}
 
+	@summary = "Default summary"
+	macro _deleteObjectDefinition(objectDefinitionName = null) {
+		Variables.assertDefined(parameterList = ${objectDefinitionName});
+
+		var baseURL = ${baseURL};
+
+		if (!(isSet(baseURL))) {
+			var baseURL = JSONCompany.getPortalURL();
+		}
+
+		var objectDefinitionId = ObjectDefinitionAPI.getObjectDefinitionIdByName(name = ${objectDefinitionName});
+		var userLoginInfo = JSONUtil2.formatJSONUser();
+
+		var curl = '''
+			${baseURL}/o/object-admin/v1.0/object-definitions/${objectDefinitionId} \
+				-u ${userLoginInfo} \
```

Comment:
oh, we use test@liferay:test before, good to know we can make this as an variable :+1: